egs/swc/s5b/local/nnet3/run_ivector_common.sh

```diff
@@ -0,0 +1,198 @@
+#!/bin/bash
```

Comment:
What is the difference between: 'local/nnet3/multi_condition/run_ivector_common.sh' and 'local/nnet3/run_ivector_common.sh'. To me it looks like a duplication of code, but if it is in other setups as well, it still can be okay.